package.json

```diff
@@ -26,15 +26,13 @@
   "devDependencies": {
     "@types/lodash": "^4.14.161",
     "@types/mathjs": "^6.0.11",
+    "estrella": "^1.4.1",
     "husky": "^5.0.0-beta.0",
     "pinst": "^2.0.0",
     "prettier": "2.1.2",
-    "ts-loader": "^8.0.4",
+    "serve-http": "^1.0.6",
     "typedoc": "^0.20.35",
-    "typescript": "^4.0.3",
-    "webpack": "^4.44.2",
-    "webpack-cli": "^3.3.12",
-    "webpack-dev-server": "^3.11.0"
+    "typescript": "^4.3.4"
```

Comment:
Please downgrade typescript version to 4.2.X in order to have compatibility with typedocs.